composer.json

```diff
@@ -28,7 +28,7 @@
         "php": ">=7.1.0"
     },
     "require-dev": {
-        "phpstan/phpstan": "1.10.39 || 1.4.10",
+        "phpstan/phpstan": "1.12.19 || 1.4.10",
```

Comment:
@PaulRotmann This is quite old, are you sure we don't want to go to 1.12.28 at least?